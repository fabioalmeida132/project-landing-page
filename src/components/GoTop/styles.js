import styled, {css} from "styled-components";

export const Container = styled.a`
  ${({theme}) => css`
      position: fixed;
      background: ${theme.colors.primaryColor};
      display: flex;
      width: 4rem;
      height: 4rem;
      align-items: center;
      justify-content: center;
      bottom: 2rem;
      right: 2rem;
      z-index: 6;
      color: ${theme.colors.white};
  `}
`;
