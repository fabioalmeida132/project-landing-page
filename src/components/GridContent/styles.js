import styled, {css} from "styled-components";

export const Container = styled.div`
  ${({theme}) => css`
    text-align: center;
    max-width: 58rem;
  `}
`;

export const Hml = styled.div`
  ${({theme}) => css`
    margin: ${theme.spacings.xhuge} 0;
  `}
`;
