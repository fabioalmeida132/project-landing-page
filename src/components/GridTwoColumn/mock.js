export default {
  title: "Grid Two Column",
  text: `Lorem ipsum dolor sit amet, consectetur adipisicing elit. Amet atque aut beatae culpa eligendi est eum
      harum, incidunt ipsam laborum nisi placeat quia quis recusandae repellat sequi, unde vel voluptas.`,
  srcImg: 'assets/images/javascript.svg',
}
