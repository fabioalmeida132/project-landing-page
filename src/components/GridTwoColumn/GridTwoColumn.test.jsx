import {screen} from "@testing-library/react";
import {renderTheme} from "../../styles/render-theme";
import mock from "./mock";
import {GridTwoColumn} from "./index";

describe("<GridTwoColumn/>", () => {
  it("should render two column grid", () => {
    const {container} = renderTheme(<GridTwoColumn {...mock}/>);
    expect(container).toMatchSnapshot();
  });
});
