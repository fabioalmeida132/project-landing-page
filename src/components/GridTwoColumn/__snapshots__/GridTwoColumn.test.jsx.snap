// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GridTwoColumn/> should render two column grid 1`] = `
.c5 {
  color: #0a1128;
  font-size: 6.4rem;
  text-transform: uppercase;
}

.c2 {
  display: grid;
  grid-template-columns: 1fr 2fr;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  gap: 3.2rem;
}

.c2 .c4 {
  margin-bottom: 4.0rem;
}

.c7 {
  width: 100%;
}

.c0 {
  background: #FFFFFF;
  color: #0a1128;
  min-height: 100vh;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c1 {
  max-width: 120rem;
  margin: 0 auto;
  padding: 3.2rem;
  width: 100%;
}

.c6 {
  font-size: 2.4rem;
}

@media (max-width:786px) {
  .c5 {
    font-size: 4.0rem;
  }
}

@media (max-width:786px) {
  .c2 {
    grid-template-columns: 1fr;
    text-align: center;
  }
}

@media (max-width:786px) {
  .c3 {
    margin-bottom: 3.2rem;
  }
}

<div>
  <div
    class="c0"
    id="id-3-3639958825459892"
  >
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        <div
          class="c3"
        >
          <h2
            class="c4 c5"
          >
            Grid Two Column
          </h2>
          <div
            class="c6"
          >
            Lorem ipsum dolor sit amet, consectetur adipisicing elit. Amet atque aut beatae culpa eligendi est eum
      harum, incidunt ipsam laborum nisi placeat quia quis recusandae repellat sequi, unde vel voluptas.
          </div>
        </div>
        <div
          class=""
        >
          <img
            class="c7"
            src="assets/images/javascript.svg"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
